state/index.js

```diff
@@ -13,16 +13,29 @@ const toDict = (array, keyFn) => array.reduce((obj, item) => ({
   [keyFn(item)]: item,
 }), {});
 
+const getCategories = badgers => (
+  Object.keys(badgers
+    .reduce((uniqueCategories, badger) => (
+      badger.categories
+        .reduce((categories, category) => (
+          categories[category] ? categories : { ...categories, [category]: 1 }
+        ), uniqueCategories)
+    ), {}))
+);
```

Comment:
Why don't you move this back into `badger-brain` under the `getBadgers` function? I.e. revert back to having specific functions there so you don't need to postprocess here. `toDict` is slightly different as it's a bit more generic.